centos/common/install_mpis.sh

```diff
@@ -113,6 +129,21 @@ setenv          MPI_MAN         /opt/intel/impi/${IMPI_2019_VERSION}/man
 setenv          MPI_HOME        /opt/intel/impi/${IMPI_2019_VERSION}/intel64
 EOF
 
+#IntelMPI-v2021
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/impi_${IMPI_2021_VERSION}
```

Comment:
Please remove IMPI 2019, and just use IMPI_VERSION as variable.

Initially it was kept different as we had to keep both IMPI 2018 and 2019.